xknx_v2.yaml

```diff
@@ -8,7 +8,10 @@ multicast_group: '224.1.2.3'
 multicast_port: 1337
 connection: # optional
   type: tunneling # or routing|auto
-  local_ip: "192.168.0.201"
+  local_ip: "192.168.111.201"
+  local_port: 42424
+  bind_ip: "192.168.0.201"
+  bind_port: 42424
```

Comment:
You'll have to add those to the schema if you want to leave it here.

https://github.com/XKNX/xknx/blob/master/xknx/config/schema.py#L79

Please also add relevant tests for it.